simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/websocket/OkHttpWebSocketClient.java

```diff
@@ -1,27 +1,41 @@
 package com.evbox.everon.ocpp.simulator.websocket;
 
+import com.evbox.everon.ocpp.simulator.configuration.SimulatorConfiguration;
+import lombok.extern.slf4j.Slf4j;
 import okhttp3.*;
 import okio.ByteString;
 
 import javax.annotation.Nullable;
+import java.util.Base64;
 import java.util.Optional;
 
+@Slf4j
 public class OkHttpWebSocketClient {
 
+    private static final String COLON = ":";
```

Comment:
this variable seems like too much, especially since it's only used in one place

Reply:
it is used in the method that can be called multiple times, as a result no need to create String-constant every time when the method is called